Blog - Google Optimize alternatives (#5136)
PostHog/posthog.com: contents/blog/google-optimize-alternatives.md

---
date: 2023-01-23
title: The best Google Optimize alternatives for apps and websites
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - andy-vandervell
featuredImage: ../images/blog/posthog-company-culture-blog.png
featuredImageType: full
category: General
tags:
  - Guides
  - Comparisons
---

> Google recently confirmed it is sunsetting Google Optimize, its A/B testing platform. In a [statement](https://support.google.com/optimize/answer/12979939) on its website, Google said:

Reviewer: ```suggestion
Google recently confirmed the sunsetting Google Optimize, its A/B testing platform. In a [statement](https://support.google.com/optimize/answer/12979939) on its website, Google said:
```
Too many "it is" in a row I think.

The statement went on to say Google is working on adding A/B testing functionality to Google Analytics 4, its somewhat divisive replacement for Google Analytics 360 (Universal Analytics). But Google **didn't commit to a launch date** for this A/B testing functionality, which suggests it may not arrive before Optimize is closed.

If this sounds like bad news to you, consider one of following six Google Optimize alternatives instead. 

> **How did choose these tools?** There are dozens of A/B testing platforms, so we tried to include a cross-section of tools designed for engineers, marketing teams, and different use cases. We prioritized tools that are self-serve (i.e. no demo or quote needed to open an account) and/or have transparent pricing.

## 1. PostHog

- **Free tier:** Yes
- **Transparent pricing:** Yes
- **Unlimited experiments:** Yes
- **A/B and multivariate tests:** Yes
- **Session recording:** Yes
- **Product analytics:** Yes
- **Feature flags:** Yes

> - **Multi-armed bandit:** No

Reviewer: Not sure what this means.

Author: Going to remove some of these

> - **Visual editor:** No

Reviewer: Also not sure what this means

- **Landing page builder:** No
- **Best for:** Product and engineering teams

> [PostHog](posthog.com/) is a comprehensive Google Optimize and [GA4 alternative](/blog/ga4-alternatives) that integrates an A/B and multivariate testing suite with event-based analytics, session recording, and feature flags.

Reviewer: ```suggestion
[PostHog](posthog.com/) is a comprehensive Google Optimize and [GA4 alternative](/blog/ga4-alternatives) that integrates A/B and multivariate testing with event-based analytics, session recording, and feature flags.
```

> PostHog experiments can be targeted by geography, cohort, and user property. It also supports minimum goals for success, secondary goals, and estimates how long it will take to get a statistically significant result. PostHog's experimentation suite is built on its feature flag functionality, making it easy to rollout winning variants.

Reviewer: ```suggestion
PostHog supports A/B and multivariate experiments that target by geography, device, source, cohort, or limitless other user properties. It tracks metric improvement goals for success, secondary goals, and estimates how long it takes to get a statistically significant result. PostHog's experimentation suite is built on its feature flag functionality, making it easy to rollout winning variants.
```

> As an all-in-one platform, PostHog simplifies your data and analytics stack by replacing multiple tools. Tight integration also makes it easy to quickly create experiments using existing cohorts in your product analytics, rather than connecting an external data source.

Reviewer: ```suggestion
As an all-in-one platform, PostHog simplifies your data and analytics stack by replacing multiple tools. This makes it easy to quickly create experiments using existing data and cohorts in your product analytics, rather than connecting an external data source.
```

> 

Reviewer: ```suggestion

PostHog has a range of [libraries and SDKs](/docs/integrate), from [JavaScript](/docs/integrate/client/js) to [Python](/docs/integrate/server/python] to [iOS](/docs/integrate/client/ios), to help integrate into your website or apps' codebase. It has has an HTML snippet for easy setup with basic sites, [Shopify](/docs/integrate/third-party/shopify), [Wordpress](/docs/integrate/third-party/wordpress), [Webflow](/tutorials/webflow), and more.

```

Unbounce offers a pleasing range of transparently-priced plans. Its entry-level 'Launch' plan is $99 per month and supports "up to 500 conversions", 20k visitors and one website domain – "conversions" being any time a user completes a conversion goal. Additional plans increase these limits and there's a 25% discount for paying annually. 

### Companies that use Unbounce

- **Packlane:** Custom packaging and boxes for retailers
- **Uberflip:** Sales engagement and lead-gen platform
- **PitchBook:** M&A database and marketing intelligence

## 3. GrowthBook

- **Free tier:** Yes
- **Transparent pricing:** Yes
- **Unlimited experiments:** Yes
- **A/B and multivariate tests:** Yes
- **Session recording:** No
- **Product analytics:** No
- **Feature flags:** Yes
- **Multi-armed bandit:** No
- **Visual editor:** Yes
- **Landing page builder:** No
- **Best for:** Product and engineering teams

[GrowthBook](www.growthbook.io/) is a popular open-source feature flagging and experimentation platform. It supports A/B and multivariate tests, and integrates with a range of popular analytics tools, such as Google Analytics (UA), Mixpanel, Matomo, and most popular data warehouses.

> It offers a basic visual editor so non-technical users can create experiments without writing code. GrowthBook's 'Guardrail' feature can also monitor for unintended second order effects that your potential change might impact, but which you're not actively testing against.

Reviewer: ```suggestion
It offers a basic visual editor so non-technical users can create experiments without writing code. GrowthBook's 'Guardrail' feature monitors for unintended second order effects of your change which you're not actively testing for.
```

### How much does GrowthBook cost?

> GorwthBook offers unlimited traffic (subject to a fair use policy), feature flags and experiments, pricing only on the number of user accounts. Its Starter plan includes three GrowthBook users. Its Pro plan adds advanced permissions, custom fields, and its visual A/B test editor, for just $20 per user, per month.

Reviewer: ```suggestion
GrowthBook offers unlimited traffic (subject to a fair use policy), feature flags, and experiments. It charges only on the number of user accounts. Its Starter plan includes three GrowthBook users. Its Pro plan adds advanced permissions, custom fields, and its visual A/B test editor, for just $20 per user, per month.
```

### Companies that use GrowthBook

- **Deezer:** Music app
- **Brilliant:** Learning platform
- **Typeform:** Form and survey platform for businesses

## 4. Instapage

- **Free tier:** No
- **Transparent pricing:** Partial
- **Unlimited experiments:** Yes
- **A/B and multivariate tests:** Yes
- **Session recording:** No
- **Product analytics:** No
- **Feature flags:** No
- **Multi-armed bandit:** No
- **Visual editor:** Yes
- **Landing page builder:** Yes
- **Best for:** Marketing teams

[Instapage](https://instapage.com/) is a marketing optimization platform built around a landing page builder and experimentation suite. It doesn't support multi-armed bandit testing like Unbounce, but it does include a range of useful features for marketing teams, such as personalized landing pages based on ad creative, AMP page support, loading speed optimizations, and dynamic text replacement for PPC campaigns.

### How much does Instapage cost?

> Instapage's self-serve plan costs $299 per month ($199 if billed annually) and includes most of its core features with no usage limits. Its upgrade plans are 'price on application' and add numerous value-add features, such as heatmaps, custom font support, and consulting services.

Reviewer: What does price on application mean?

Author: You have to email them for a quote / get a demo. Might come up with a better way to say this.

Reviewer: "Its upgrade plans require a sales call, but add numerous..."

### Companies that use Instapage

- **HelloFresh:** Recipe box delivery
- **Honey:** Coupon and shopping browser extension
- **Skyscanner:** Airline price comparison

> ## 5. VWO Testing

Reviewer: ```suggestion
## 5. VWO
```

Reviewer: If they want to name themselves something unclear, we should let them 😃

Author: Haha, in this case I wanted to specify this specific 'product' they have as they have a bunch of different ones, some of which also include A/B testing. It's, frankly, baffling.

- **Free tier:** Yes
- **Transparent pricing:** No
- **Unlimited experiments:** Yes
- **A/B and multivariate tests:** Yes
- **Session recording:** No
- **Product analytics:** No
- **Feature flags:** No
- **Multi-armed bandit:** No
- **Visual editor:** Yes
- **Landing page builder:** No
- **Best for:** Marketing teams

> [VWO](vwo.com) is a testing platform that primarily targets large enterprises. Its experimentation platform is pretty comprehensive, including support for A/B and multivariate tests, and a visual editor, though some features are locked behind its numerous product tiers. Geo targeting, for example, is only available on the 'Pro' tier of VWO Testing, while feature flag support is only available as part of its VWO FullStack product.

Reviewer: ```suggestion
[VWO](vwo.com) is a testing platform that primarily targets large enterprises. Its experimentation platform includes support for A/B and multivariate tests, and a visual editor, though some features are locked behind its numerous product tiers. Geo-targeting, for example, is only available on the 'Pro' tier of VWO Testing, while feature flags are only available as part of its VWO FullStack product.
```

### How much does VWO cost?

Outside of its new Starter plan, VWO is largely a 'price on application' product. Its Starter plan is free up to 50k visitors per month, and $199 per month up to 100k visitors per month, but it's quite limited. It only supports basic A\B tests and split URL testing, and you can only target based on device.

### Companies that use VWO

- **Chargebee:** Payments and subscription platform  
- **Amway:** Mutli-level marketing
- **TMZ:** Celebrity and entertainment news

## 6. LaunchDarkly

- **Free tier:** Yes
- **Transparent pricing:** Partial
- **Unlimited experiments:** No
- **A/B and multivariate tests:** Yes
- **Session recording:** No
- **Product analytics:** No
- **Feature flags:** Yes
- **Multi-armed bandit:** No
- **Visual editor:** No
- **Landing page builder:** No
- **Best for:** Engineering and product teams

> [LaunchDarkly](launchdarkly.com) is a leading platform for feature management (feature flags) that also enables experimentation. Like all the products in this list, it supports A/B and multivariate testing. In contrast to Unbounce, however, LaunchDarkly is designed exclusively for product and engineering teams, so it lacks marketing-friendly features like a visual editor. 

Reviewer: ```suggestion
[LaunchDarkly](launchdarkly.com) is a leading platform for feature management (feature flags) that also enables experimentation. Like all the products in this list, it supports A/B and multivariate testing. In contrast to Unbounce, however, LaunchDarkly is designed for product and engineering teams, so it lacks marketing-friendly features like a visual editor. 
```
